### PR TITLE
Creating validators

### PR DESCRIPTION
**Pull request by demakk**

### CODE REVIEW

File: Backend/JourneyLog.BLL/Validators/Auth/SignupModelValidator.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        RuleFor(info => info.FirstName)
+            .NotEmpty().WithMessage("First name cannot be empty, null or whitespace")
+            .MinimumLength(3).WithMessage("First name must be at least 3 characters long")
+            .MaximumLength(50).WithMessage("Last name must be at most 50 characters long");
+        
+        RuleFor(info => info.LastName)
+            .NotEmpty().WithMessage("First name cannot be empty, null or whitespace")
+            .MinimumLength(3).WithMessage("Last name must be at least 3 characters long")
+            .MaximumLength(50).WithMessage("Last name must be at most 50 characters long");
```
Comment (techcat18) [inline Backend/JourneyLog.BLL/Validators/Auth/SignupModelValidator.cs]: Chinese names triggered

File: Backend/JourneyLog.BLL/Validators/NotePhotos/CreateNotePhotoModelValidator.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+using JourneyLog.BLL.Models.TravelLogPlaceNotePhotos;
+
+namespace JourneyLog.BLL.Validators.NotePhotos;
+
+public class CreateNotePhotoModelValidator : AbstractValidator<CreateNotePhotoModel>
+{
+    public CreateNotePhotoModelValidator()
+    {
+        RuleFor(e => e.ImageBase64)
+            .NotEmpty().WithMessage("Image link cannot be empty, null or whitespace");
```
Comment (techcat18) [inline Backend/JourneyLog.BLL/Validators/NotePhotos/CreateNotePhotoModelValidator.cs:11]: shouldn't be image base 64..

File: Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceRatingModelValidator.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+namespace JourneyLog.BLL.Validators.Place;
+
+public class CreateUpdatePlaceRatingModelValidator : AbstractValidator<CreateUpdatePlaceRating>
+{
+    public CreateUpdatePlaceRatingModelValidator()
+    {
+        RuleFor(e => e.Rating)
+            .NotEmpty().WithMessage("Rating value cannot be null, empty or whitespace")
+            .InclusiveBetween(1, 5).WithMessage("Rating value can only be in range between 1 and 5 inclusive");
```
Comment (techcat18) [inline Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceRatingModelValidator.cs]: 1-10

File: Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceReviewModelValidator.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+using JourneyLog.BLL.Models.Place;
+
+namespace JourneyLog.BLL.Validators.Place;
+
+public class CreateUpdatePlaceReviewModelValidator : AbstractValidator<CreateUpdatePlaceReview>
+{
+    public CreateUpdatePlaceReviewModelValidator()
+    {
+        RuleFor(e => e.Review)
+            .NotEmpty().WithMessage("Rating value cannot be null, empty or whitespace")
```
Comment (techcat18) [inline Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceReviewModelValidator.cs]: Review not rating

File: Backend/JourneyLog.BLL/Validators/Place/GetPlaceByRadiusModelValidator.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public GetPlaceByRadiusModelValidator()
+    {
+        RuleFor(e => e.lat)
+            .NotEmpty().WithMessage("Latitude cannot be null or empty");
+        RuleFor(e => e.lon)
+            .NotEmpty().WithMessage("Longitude cannot be null or empty");
+        RuleFor(e => e.radius)
+            .NotEmpty().WithMessage("Radius cannot be null or empty")
+            .InclusiveBetween(1, 20000).WithMessage("Radius can only be in range between 1 and 20 000");
```
Comment (techcat18) [inline Backend/JourneyLog.BLL/Validators/Place/GetPlaceByRadiusModelValidator.cs]: I don't think there should upper limit for radius

Comment (NMishchenko) [reply Backend/JourneyLog.BLL/Validators/Place/GetPlaceByRadiusModelValidator.cs]: 100 000 will be ok

File: Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceRatingModelValidator.cs (hunk 3 above)
Comment (NMishchenko) [reply Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceRatingModelValidator.cs]: 1-3 :c

File: Backend/JourneyLog.BLL/Validators/Auth/SignupModelValidator.cs (hunk 1 above)
Comment (demakk) [reply Backend/JourneyLog.BLL/Validators/Auth/SignupModelValidator.cs]: Done

File: Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceReviewModelValidator.cs (hunk 4 above)
Comment (demakk) [reply Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceReviewModelValidator.cs]: Done

File: Backend/JourneyLog.BLL/Validators/Place/GetPlaceByRadiusModelValidator.cs (hunk 5 above)
Comment (demakk) [reply Backend/JourneyLog.BLL/Validators/Place/GetPlaceByRadiusModelValidator.cs]: Resolved

File: Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceRatingModelValidator.cs (hunk 3 above)
Comment (demakk) [reply Backend/JourneyLog.BLL/Validators/Place/CreateUpdatePlaceRatingModelValidator.cs]: Done

**Pull request merged**